acceptance/suites/tests/certificate_authority/intermediate_ca.rb

```diff
@@ -88,14 +91,15 @@ def fixture(filename)
     on master, "service #{master['puppetservice']} start"
   end
 
-  test_agent = (agents - [master]).first
 
   # Remove the old CA infrastructure from an agent
-  on test_agent, "rm -rf #{test_agent.puppet['ssldir']}"
+  ssldir = puppet_config(test_agent, 'ssldir', section: 'agent')
+  cert = puppet_config(master, 'hostcert', section: 'master')
+  cacert = puppet_config(master, 'localcacert', section: 'master')
+  key = puppet_config(master, 'hostprivkey', section: 'master')
 
-  ssl_options = "--cert #{master.puppet['hostcert']} " +
-                "--cacert #{master.puppet['localcacert']} " +
-                "--key #{master.puppet['hostprivkey']}"
+  on test_agent, "rm -rf #{ssldir}"
+  ssl_options = "--cert #{cert} --cacert #{cacert} --key #{key}"
 
   # Create a dummy agent
```

Comment:
Would you mind making this and the other comments in this file into calls to `step`? It makes the output much easier to parse.